test/e2e/persistent_volumes.go

```diff
@@ -164,40 +204,44 @@ func makePersistentVolumeClaim(ns string) *api.PersistentVolumeClaim {
 	}
 }
 
-func makeCheckPod(ns string, nfsserver string) *api.Pod {
+func makeCheckPod(ns string, pvcName string) *api.Pod {
 	// Prepare pod that mounts the NFS volume again and
 	// checks that /mnt/index.html was scrubbed there
+
+	var isPrivileged bool = true
 	return &api.Pod{
 		TypeMeta: unversioned.TypeMeta{
 			Kind:       "Pod",
 			APIVersion: testapi.Default.GroupVersion().String(),
 		},
 		ObjectMeta: api.ObjectMeta{
-			GenerateName: "checker-",
+			GenerateName: "rw-pod-",
 			Namespace:    ns,
 		},
 		Spec: api.PodSpec{
 			Containers: []api.Container{
 				{
-					Name:    "scrub-checker",
+					Name:    "rw-pod",
 					Image:   "gcr.io/google_containers/busybox:1.24",
 					Command: []string{"/bin/sh"},
-					Args:    []string{"-c", "test ! -e /mnt/index.html || exit 1"},
+					Args:    []string{"-c", "touch /mnt/SUCCESS && cat /mnt/index.html && exit 0 || exit 1"},
 					VolumeMounts: []api.VolumeMount{
 						{
-							Name:      "nfs-volume",
+							Name:      "nfs-pvc",
 							MountPath: "/mnt",
 						},
 					},
+					SecurityContext: &api.SecurityContext{
+						Privileged: &isPrivileged,
```

Comment:
I assume the pod needs to be priv to write to the mount. However, in general, we don't want to encourage priv pods, so maybe you can change the perms on the mount? The nfs-server pod creates the export on /:
```
#nfs-server pod:
sh-4.2# cat /etc/exports               
/exports *(rw,fsid=0,insecure,no_root_squash)
/ *(rw,fsid=0,insecure,no_root_squash)
```
and the nfs pv created above sets the vol's path to "/". Perm's on the nfs-server's "/" are:
```
drwxr-xr-x. 18 root root 4096 Jun  9 20:26 /
```
so non-owner(root) can only read, not write.
This is mostly rambling and I can't provide a better solution than *priv* until I am able to get the checker pod running in my env...